ruby/ext/google/protobuf_c/encode_decode.c

```diff
@@ -613,6 +688,23 @@ static void add_handlers_for_oneof_field(upb_handlers *h,
   upb_handlerattr_uninit(&attr);
 }
 
+static bool add_unknown_handler(void* closure, const void* hd, const char* buf,
```

Comment:
The name of this function is a little confusing. The other functions that start with `add_` add a handler, but I think this function *is* a handler. Maybe call it `unknown_field_handler`?

Reply:
done